python/external_config/process_execution.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) 2018 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
```

Comment:
note: this was carried over from the server fw (which carried it over from the desktop engine). We may not need this in the future at all, but it's there for the time being.